### ISSUE TITLE
return type this throws TypeError

### ISSUE THREAD

**Issue by joebowbeer** [Khan/flow-to-ts#131]
The following code throws a TypeError:

```
// @flow
export default class MyClass {
  foo(): this {
    return this
  }
}
```

```
TypeError: Property typeAnnotation of TSTypeAnnotation
expected node to be of a type ["TSType"]
but instead got "ThisTypeAnnotation"
```

Changing `this` to `MyClass` fixes it.

Playground:

https://flow-to-ts.netlify.com/#code=Ly8gQGZsb3cKZXhwb3J0IGRlZmF1bHQgY2xhc3MgTXlDbGFzcyB7CiAgZm9vKCk6IHRoaXMgewogICAgcmV0dXJuIHRoaXMKICB9Cn0K&prettier=1&semi=1&singleQuote=0&tabWidth=4&trailingComma=all&bracketSpacing=0&arrowParens=avoid&printWidth=80&inlineUtilityTypes=0

**Comment by k4b7** [comment]
I didn't realize that `this` could be used as a return type in Flow.  It looks like the same is true in TypeScript.

**Comment by k4b7** [comment]
I've published [v0.2.1](https://github.com/Khan/flow-to-ts/releases/tag/v0.2.1) which should solve this.  Please let me know if you're still experiencing issues with this.
